src/actions/get-bounty-canceled-event.ts

```diff
@@ -10,14 +10,14 @@ import logger from "src/utils/logger-handler";
 import { slashSplit } from "src/utils/string";
 
 export const name = "getBountyCanceledEvents";
-export const schedule = "1 * * * * *";
-export const description = "retrieving bounty canceled events";
+export const schedule = "*/30 * * * * *"; // Each 10 minuts
```

Comment:
Should this action be executed every 30 seconds or minutes ? I think `*/30 * * * * *` means every 30 seconds

Reply:
you right, i will fix it